app/controllers/users/induction_coordinator_registrations_controller.rb

```diff
@@ -1,16 +1,18 @@
 # frozen_string_literal: true
 
-class Users::InductionCoordinators::RegistrationsController < Devise::RegistrationsController
-  def start_registration; end
+class Users::InductionCoordinatorRegistrationsController < Devise::RegistrationsController
```

Comment:
After the discussion with Alaric it looks like only induction_coordinators will be able to register through the platform. In that case I'm thinking I'll replace Users::InductionCoordinatorRegistrationsController with just Users::RegistrationsController

Reply:
That would be good